slidechain/export.go

```diff
@@ -106,28 +162,43 @@ func (c *Custodian) pegOutFromExports(ctx context.Context) {
 			log.Printf("pegging out export %x: %d of %s to %s", txid, amounts[i], asset.String(), exporters[i])
 
 			peggedOut := pegOutOK
-			err = c.pegOut(ctx, exporter, asset, int64(amounts[i]), tempID, xdr.SequenceNumber(seqnums[i]))
+			err = c.pegOut(ctx, exporter, asset, amounts[i], tempID, xdr.SequenceNumber(seqnums[i]))
 			if err != nil {
 				peggedOut = pegOutFail
 				if herr, ok := errors.Root(err).(*horizon.Error); ok {
 					resultCodes, err := herr.ResultCodes()
 					if err != nil {
-						log.Fatalf("getting error codes from failed submission of tx %s", txid)
+						log.Fatalf("getting error codes from failed submission of tx %x", txid)
```

Comment:
Include `herr` and/or `err` in this log line. And what about the outermost `err` (which is being shadowed)?